packages/core/src/App/Constants/routes-config.js

```diff
@@ -190,6 +190,7 @@ const initRoutesConfig = () => [
     { path: routes.endpoint, component: Endpoint, title: 'Endpoint' }, // doesn't need localization as it's for internal use
     { path: routes.redirect, component: Redirect, title: localize('Redirect') },
     {
+        counter: <CashierNotifications />,
```

Comment:
Do we need this as route config? This config prop may not be used for other routes.
Can we make it as `renderTitle` callback function?

Reply:
Hmm, can you elaborate?

Reply:
You can have a config prop named `renderTitle`. When invoked it should return a view. In your case, you can return a view with title and counter element. We use this kind of approach for rendering modal title. [reference](https://github.com/binary-com/deriv-app/blob/7c1120242b2e159a2bb0b878f129b44438e66ee4/packages/components/src/components/modal/modal.jsx#L61)

Reply:
Ah that makes sense 👍